solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -217,11 +220,11 @@ function update_solution_config(){
 
 function add_image_info() {
     add_secondary_separator "Updating cortx images info in solution.yaml"   
-    update_image control-pod $CORTX_CONTROL_IMAGE
-    update_image data-pod $CORTX_DATA_IMAGE
-    update_image server-pod $CORTX_SERVER_IMAGE
-    update_image ha-pod $CORTX_CONTROL_IMAGE
-    update_image client-pod $CORTX_DATA_IMAGE
+    update_image control-pod "$CORTX_ACTUAL_CONTROL_IMAGE"
```

Comment:
Can't  we use `update_image control-pod "$CORTX_SERVER_IMAGE"` directly here

Reply:
Done and tested